orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -1405,4 +1407,40 @@ def testORTTrainerRunSymbolicShapeInfer():
 
     # Compare losses
     _test_helpers.assert_model_outputs(new_loss, expected_loss)
-    _test_helpers.assert_model_outputs(legacy_loss, expected_loss)
\ No newline at end of file
+    _test_helpers.assert_model_outputs(legacy_loss, expected_loss)
+
+
+@pytest.mark.parametrize("device", ["cpu", "cuda"])
```

Comment:
Could you parameterize model_with_training_graph_path so that you test it for both empty and valid path and then test if the file doesnt or do exist, depending on the flag?